src/components/ChangeUser.jsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState, useContext } from "react";
+import { UserContext } from "../contexts/User";
+import axios from "axios";
+//Here we need the context and we're gonna need to useContext
+//This is also where we need out api helper function
+
+function ChangeUser() {
+  const [users, setUsers] = useState([]);
+  //New state that is all possible user profiles
+  const [isLoading, setIsLoading] = useState(true);
+  const { user, setUser } = useContext(UserContext);
+  //Grab and use context and destructure the setUser function part only
+  const usersCopy = [...users];
+
+  function displayUsers(username) {
```

Comment:
```js
  function displayUsers(username) {
```
Don't think you ever use this function ^ Go ahead and remove it

Reply:
removed